auth-api/src/auth_api/schemas/org.py

```diff
@@ -27,7 +27,7 @@ class Meta:  # pylint: disable=too-few-public-methods
         """Maps all of the Org fields to a default schema."""
 
         model = OrgModel
-        exclude = ('members', 'contacts', 'invitations', 'affiliated_entities')
+        exclude = ('members', 'contacts', 'invitations', 'affiliated_entities', 'suspension_reason')
```

Comment:
Any reason to exclude this field? Are you still be able to include suspension reason in org response?

Reply:
Yeah. It is a relationship that needs to be excluded. I have the attribute suspension_reason_code that has the valkue  :) 
![image](https://user-images.githubusercontent.com/78374865/109725331-d3303f80-7b65-11eb-9199-aad84003bf90.png)